### PR TITLE
Blockly python switch fix

### LINKED ISSUE #4: Blockly code removed when switching between python/blockly

### LINKED ISSUE #10: Block removed when moving block onto flyout section

### PR DESCRIPTION
**Pull request by mklomp**
Fixing #4 and #10

### CODE REVIEW

File: vue-frontend/locales/nl.json [hunk 1]
```diff
@@ -42,6 +42,8 @@
     "move_stop": "Stop"
   },
   "programming": {
+    "python": "python",
+    "blockly": "blokken",
```
Comment (ArendJan) [inline vue-frontend/locales/nl.json:46]: I'd keep it as Blockly, as that is the programming language, maybe capitalize python as well

File: vue-frontend/src/components/Blockly.vue [hunk 2]
```diff
@@ -510,7 +516,7 @@ export default {
 
      params.get((res) => {
        this.params = res;
-       setTimeout(this.load_blockly, 500);
+       setTimeout(this.load_blockly, 10);
```
Comment (ArendJan) [inline vue-frontend/src/components/Blockly.vue:519]: Maybe change this to setImmediate, or Vue.nextTick()

File: vue-frontend/src/views/Programming.vue [hunk 3]
```diff
@@ -82,6 +102,23 @@ export default {
...
+        },
+  },
+  computed: {
+       isBlockly: function(){
+           return this.language == "blockly";
+       },
   }
+
+
+
```
Comment (ArendJan) [inline vue-frontend/src/views/Programming.vue:121]: Some extra newlines, will be removed when we merge the workflows

File: vue-frontend/src/components/Xterm.vue [hunk 4]
```diff
@@ -35,14 +35,15 @@ export default {
...
-                  this.linenr_socket.send("c");
-                  this.$store.dispatch('setExecution', 'running');
-              };
-
               this.linenr_socket.onmessage = (event) => {
                 if (event.data != 0) {
                   // Update only when in step/pause mode
+                  if (event.data.substr(0, 4) == "pid:"){
+                     let debugger_pid = String(event.data.substr(4));
+let strace_cmd = 'strace -ff -e write=1,2 -s 1024 -p ' + debugger_pid + ' 2>&1 | grep "^ |" --line-buffered | stdbuf -oL cut -b11-60 | stdbuf -oL sed -e "s/ //g" | xxd -r -p';
```
Comment (ArendJan) [inline vue-frontend/src/components/Xterm.vue:43]:
```suggestion
                     let strace_cmd = 'strace -ff -e write=1,2 -s 1024 -p ' + debugger_pid + ' 2>&1 | grep "^ |" --line-buffered | stdbuf -oL cut -b11-60 | stdbuf -oL sed -e "s/ //g" | xxd -r -p';
```

File: vue-frontend/locales/nl.json (hunk 1 above)
Comment (mklomp) [reply vue-frontend/locales/nl.json:46]: Usually on primary-/highschools they refer to block and typed programmig laguages. I did add a capital (as discussed)

File: vue-frontend/src/components/Blockly.vue (hunk 2 above)
Comment (mklomp) [reply vue-frontend/src/components/Blockly.vue:541]: Tried Vue.nextTick(), but did not work. Maybe we an have a look at that at a later stage.

**Pull request merged**